apps/services/auth/ids-api/infra/ids-api.ts

```diff
@@ -100,6 +100,11 @@ export const serviceSetup = (): ServiceBuilder<'services-auth-ids-api'> => {
         staging: 'digitaliceland',
         dev: 'digitaliceland',
       },
+      ALSO_USE_FAKE_USER_API: {
+        dev: 'true',
+        staging: 'false',
+        prod: 'false',
```

Comment:
How will this affect the fake user login in production used for the app review, where we enable a single fake user with random phone number using the env variable?

Reply:
The idea is that these changes should not affect production at all.

In the env var we can also provide the name of the fake user and it is used as fallback.